modulefiles/wcoss_dell_p3/fv3

```diff
@@ -17,6 +17,7 @@ module-whatis "loads NEMS FV3 prerequisites on Venus and Mars"
 ##
```

Comment:
I guess we need to add these CMAKE_... stanzas for linux.gnu, linux.intel and macos.gnu as well?

Reply:
Since we can not assume that on generic linux machine (ie. laptop/desktop) modules are available, nor location of the libraries or compiler (and mpi wrapper) names, I'm not loading (sourcing) these files. Build scripts (build.sh) sets up all required environment variables.

Reply:
Got it - makes total sense. Thanks for the clarification.